src/static/riot/competitions/form/_leaderboard.tag

```diff
@@ -252,16 +258,16 @@
                 <td each="{ column, index in columns }" class="center aligned">
                     <div class="ui field">
                         <select class="ui fluid small dropdown" onchange="{ edit_column_type.bind(this, index) }">
-                            <option selected>
+                            <option selected="{!column.computation}">
                                 ------
                             </option>
-                            <option>Average</option>
+                            <option value="avg" selected="{column.computation === 'avg'}">Average</option>
                         </select>
```

Comment:
Are there more computation options that need to get added here?

Reply:
I don't know where they're even defined in v1.5, we can stick with just this for now.